integrasjontjenester/dokumentbestiller/src/main/java/no/nav/foreldrepenger/tilbakekreving/dokumentbestilling/felles/pdf/JournalføringTjeneste.java

```diff
@@ -92,9 +99,10 @@ private AvsenderMottaker lagMottakerVerge(Long behandlingId) {
                         .medTittel(brevMetadata.getTittel())
                         .medBrevkode(brevMetadata.getFagsaktype().getKode() + "-TILB")
                         .leggTilDokumentvariant(new Dokumentvariant(Dokumentvariant.Variantformat.ARKIV, Dokumentvariant.Filtype.PDFA, vedleggPdf)))
+                .medEksternReferanseId(callId)
```

Comment:
Vil callid være unik for journalposten?

Reply:
Et alternativ er å legge inn en UUID som task_parameter brevUuid når du oppretter tasken som kaller journalførUtgåendeBrev.
Dokarkiv-klient skal håndtere 409-konvensjonen og returnere journalpostid.
Prosesstasks vil alltid ha satt en callId - hvis den er tom lages den på formal CallId_<tid>_<random>. se TaskManagerRunnableTask